VCEntities/VCEntities/oidc/PresentationRequestClaims.swift

```diff
@@ -74,47 +77,7 @@ public struct PresentationRequestClaims: OIDCClaims, Equatable {
         self.idTokenHint = idTokenHint
         self.iat = iat
         self.exp = exp
-    }
-    
-    public init(from decoder: Decoder) throws {
```

Comment:
just curious, what is the encode and decode code below?

Reply:
it helps with serialization but with this new change, we can just the default methods instead of overriding them.